### ISSUE TITLE
Just wondering - DataLoader does not shuffle in the example

### ISSUE THREAD

**Issue by daisukelab** [lucidrains/byol-pytorch#23]
https://github.com/lucidrains/byol-pytorch/blob/master/examples/lightning/train.py#L95

Hi @lucidrains thanks for sharing great implementation.
Just wondering if we don't really need `shuffle=True` with `train_loader`.

> train_loader = DataLoader(ds, batch_size=BATCH_SIZE, num_workers=NUM_WORKERS)

If we use a dataset big and random enough, this mightn't be a problem, but would makes trouble with smaller set.

Just close this if this is expected or you think this is not a problem.
Thanks anyway.

**Comment by lucidrains** [comment]
done! https://github.com/lucidrains/byol-pytorch/commit/a396acce6c4acb7f2038ba22020bfa44bcd9725f
